doc/source/ray-air/doc_code/air_ingest.py

```diff
@@ -139,3 +139,71 @@ def make_ds_2():
 # -> {'train': DatasetConfig(fit=True, split=True, ...),
 #     'side': DatasetConfig(fit=False, split=False, transform=False, ...), ...}
 # __config_2_end__
+
+# __config_4__
+import ray
+from ray import train
+from ray.data import Dataset
+from ray.air.train.data_parallel_trainer import DataParallelTrainer
+from ray.air.config import DatasetConfig
+
+
+def train_loop_per_worker():
+    # By default, bulk loading is used and returns a Dataset object.
+    data_shard: Dataset = train.get_dataset_shard("train")
+
+    # Manually iterate over the data 10 times (10 epochs).
+    for _ in range(10):
+        for batch in data_shard.iter_batches():
+            print("Do some training on batch", batch)
+
+    # View the stats for performance debugging.
+    print(data_shard.stats())
+
+
+my_trainer = DataParallelTrainer(
```

Comment:
`DataParallelTrainer` is a developer API so might be confusing to users to show this as an example. For these examples could we show usage with or TorchTrainer or TFTrainer?

Reply:
I changed it to TorchTrainer, but decline to show to_torch/to_tf. Those APIs are too framework specific, and also a little half baked right now.